src/models/user.ts

```diff
@@ -0,0 +1,92 @@
+// Import the necessary modules
+import mongoose, { Document, Schema } from 'mongoose';
+
+// Define the user schema
+interface UserSchema extends Document {
```

Comment:
Rename userSchema `IUser`
and export it
`export interface IUser extends Document {`